src/carnot/funcs/protocols/mux.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include <string>
+
+namespace px {
+namespace carnot {
+namespace funcs {
+namespace protocols {
+namespace mux {
+
+inline std::string FrameTypeName(int frame_type) {
+  switch (frame_type) {
+    case 1:
+      return "Treq";
+    case -1:
+      return "Rreq";
+    case 2:
+      return "Tdispatch";
+    case -2:
+      return "Rdispatch";
+    case 64:
+      return "Tdrain";
+    case -64:
+      return "Rdrain";
+    case 65:
+      return "Tping";
+    case -65:
+      return "Rping";
+    case 66:
+      return "Tdiscarded";
+    case -66:
+      return "Rdiscarded";
+    case 67:
+      return "Tlease";
+    case 68:
+      return "Tinit";
+    case -68:
+      return "Rinit";
+    case -128:
+      return "Rerr";
+    case 127:
+      return "Rerr (legacy)";
+    case -62:
+      return "Tdiscarded (legacy)";
+    default:
+      return std::to_string(frame_type);
```

Comment:
Can we use the absl functions? std::string functions generally look at locale and we avoid using them in carnot. We might also want to print it as `absl::StringFormat("Unknown (%d), frame_type);`.

Reply:
For sure. This was copy and pasted from the amqp example, but it seems a few others are using that as well.

```
[tw-mbp-ddelnano pixie (ddelnano/add-mux-carnot-func-for-mux-type-translation)]% git grep std::to_string src/carnot/funcs/protocols
src/carnot/funcs/protocols/amqp.h:      return std::to_string(frame_type);
src/carnot/funcs/protocols/kafka.h:      return std::to_string(api_key);
src/carnot/funcs/protocols/mysql.h:      return std::to_string(cmd);
```

mux's case was updated in the latest revision.

Reply:
We should fix the other ones as well, but that can be done separately.